game-core/src/main/java/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -122,34 +123,34 @@ private void exportXmlFile() {
     }
   }
 
-  private void addSaveScreenshot() {
-    final Action abstractAction =
-        SwingAction.of(
-            "Export Map Snapshot",
-            e -> {
-              // get current history node. if we are in history view, get the selected node.
-              final HistoryPanel historyPanel = frame.getHistoryPanel();
-              final HistoryNode curNode;
-              if (historyPanel == null) {
-                curNode = gameData.getHistory().getLastNode();
-              } else {
-                curNode = historyPanel.getCurrentNode();
-              }
-              ScreenshotExporter.exportScreenshot(frame, gameData, curNode);
-            });
-    add(abstractAction).setMnemonic(KeyEvent.VK_E);
+  private JMenuItem createSaveScreenshotMenu() {
+    return new JMenuItemBuilder("Export Map Snapshot", KeyCode.E)
```

Comment:
```suggestion
    return new JMenuItemBuilder("Export Map Screenshot", KeyCode.E)
```
Out of scope really, but this naming difference bothers me for a while now.